templates/web/base/hercules/bin_days.html

```diff
@@ -0,0 +1,76 @@
+[% SET bodyclass = 'hercules' %]
+[% USE date(format = '%A, %-d~~~ %B') %]
+
+[% INCLUDE header.html %]
+
+<h1 class="govuk-heading-xl">Your bin days</h1>
+
+<dl class="hercules__address">
+  <dt class="hercules__address__title">Address</dt>
+  <dd class="hercules__address__property">[% property.Description | title %]</dd>
+</dl>
+<div class="hercules__collections">
+  <h2 class="govuk-heading-l">Your collections</h2>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">  
+
+[% FOR unit IN service_data %]
```

Comment:
This could do with some handling for situations when they have no collections. E.g the PO BOX that is at BR1 3UN has no collections but you can still click report missed collection.

Reply:
Possibly there should be an entirely separate page for "we don't collect stuff from here".